runtime/doc/repeat.txt

```diff
@@ -231,6 +231,10 @@ For writing a Vim script, see chapter 41 of the user manual |usr_41.txt|.
 				:runtime plugin/**/*.vim
 <			would source the first file only.
 
+			For each {file} pattern, if a file has both a `.vim`
+			and `.lua` extensions, the `.vim` version will take
+			priority and be sourced first.
```

Comment:
```suggestion
			and `.lua` extensions, the `.vim` version will be sourced first.
```

"Priority" is whatever is sourced last, right? So maybe it's less confusing to not mention priority here.